### ISSUE TITLE
Metrics overview page seems to miss some instruments

### ISSUE THREAD

**Issue by pirgeo** [open-telemetry/opentelemetry.io#2492]
**What needs to be changed?** 
The[ metrics overview page](https://opentelemetry.io/docs/concepts/signals/metrics/) describes three metrics instruments (counter, measure, and gauge). The [specification](https://opentelemetry.io/docs/reference/specification/metrics/api/) describes Counter, UpDownCounter, Histogram, Gauge, and asynchronous versions of some of these. Am I missing something on the overview page, or is it simply outdated?

**What is the name + path of the page that needs changed?** 
* https://github.com/open-telemetry/opentelemetry.io/blob/main/content/en/docs/concepts/signals/metrics.md
* https://opentelemetry.io/docs/concepts/signals/metrics/

**Additional context**: 
Metrics specification: https://opentelemetry.io/docs/reference/specification/metrics/api/

**Comment by svrnm** [comment]
yes, this is missing from the docs, feel free to provide a PR to fix this.

**Comment by pirgeo** [comment]
In https://github.com/open-telemetry/opentelemetry.io/pull/2493, I updated a couple of details about that page

**Comment by svrnm** [comment]
TY!

**Comment by cartermp** [comment]
Thanks @pirgeo -- the page could use a lot more love, yeah. Your PR helps get us where we need to be.
